dictionary/index.js

```
@@ -0,0 +1,31 @@
+ */
+
+const Validation = require('../src/utils/Validation');
+
+const locales = {
+    wallet: require('./wallet.local'),
+    common: require('./common.local'),
+}
+
+const setLocale = (provider) => {
+    if (Validation.isServiceProvider(provider)) {
+        locales[provider] = require(__dirname + '../dictionary/' + provider + '.local');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)